db/re/item_db_usable.yml

```
@@ -62369,7 +62369,7 @@ Body:
     NoUse:
       Sitting: true
     Script: |
-      /* TODO pet 1782; */
+      /* TODO pet; */
```

```suggestion
      /* TODO pet 1782; */
```
Otherwise the important information may be lost.